resources/eds_templates/diff_report/templates/layout.html

```diff
@@ -9,68 +9,123 @@
     <meta name=viewport content="width=device-width, initial-scale=1">
     <meta name="mobile-web-app-capable" content="yes">
 
-    <!-- build:css styles/vendor.css -->
-    <!-- bower:css -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
-    <!-- endbower -->
-    <!-- endbuild -->
-    <link href='//fonts.googleapis.com/css?family=Open+Sans:400,700,300&subset=latin,vietnamese' rel='stylesheet'
-          type='text/css'>
+
+    <link rel="stylesheet" type="text/css"
+          href="https://cdnjs.cloudflare.com/ajax/libs/fomantic-ui/2.8.7/semantic.min.css">
     <link href="https://cdnjs.cloudflare.com/ajax/libs/jquery.tocify/1.9.0/stylesheets/jquery.tocify.min.css"
           rel="stylesheet"
           type="text/css">
-    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
-            rel="stylesheet"
-            type="text/css">
-
     <link href="https://code.jquery.com/ui/1.11.4/themes/redmond/jquery-ui.css" rel="stylesheet" type="text/css">
-    <!-- build:css styles/main.css -->
-    {#    <link rel="stylesheet" href="main.css">#}
-    <!-- endbuild -->
+    <link href="https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css" rel="stylesheet" type="text/css">
+
+
+    <style>
+        .buttons-print {
+            margin-bottom: 0.5em !important;
+        }
+
+        #toc {
+            width: 15%;
```

Comment:
you should put the `css` into this [file](https://github.com/meaningfy-ws/rdf-differ/blob/master/rdf_differ/entrypoints/ui/static/main.css)

Reply:
maybe in the future when eugen wants to